core/src/test/java/io/github/wysohn/triggerreactor/core/main/TriggerReactorCoreTest.java

```diff
@@ -936,6 +936,36 @@ public void command_invTrigger_item() throws Exception {
         assertEquals("STONE", inventoryTriggerManager.get("MyInventory").getItems()[0].getType());
     }
 
+    @Test
+    public void command_invTrigger_canPickupItem() throws Exception {
+        // arrange
+        UUID uuid = UUID.randomUUID();
+        IPlayer sender = mock(IPlayer.class);
+        Injector injector = createInjector();
+        TRGCommandHandler handler = injector.getInstance(TRGCommandHandler.class);
+
+        InventoryTriggerManager inventoryTriggerManager = injector.getInstance(InventoryTriggerManager.class);
+
+        when(sender.getUniqueId()).thenReturn(uuid);
+        when(sender.hasPermission(TRGCommandHandler.PERMISSION)).thenReturn(true);
+        when(pluginManagement.isEnabled()).thenReturn(true);
+
+        // act
+        handler.onCommand(sender, COMMAND_NAME, new String[]{"i", "MyInventory", "create", "54", "#MESSAGE \"Hello World\""});
+
+        // assert
+        assertNotNull(inventoryTriggerManager.get("MyInventory"));
+        assertEquals(54, inventoryTriggerManager.get("MyInventory").getItems().length);
+        assertEquals("#MESSAGE \"Hello World\"", inventoryTriggerManager.get("MyInventory").getScript());
+        assertEquals(false, inventoryTriggerManager.get("MyInventory").canPickup());
+
+        // act 2
```

Comment:
I'd suggest separating `act 2` and `assert 2` into their own folder.

I know it's repetitive, but we could follow the best practices: https://medium.com/@tasdikrahman/f-i-r-s-t-principles-of-testing-1a497acda8d6

of course you can copy the arrange part